src/lib/application/application.factory.ts

```diff
@@ -42,10 +42,14 @@ function transform(options: ApplicationOptions): ApplicationOptions {
   target.language = !!target.language ? target.language : DEFAULT_LANGUAGE;
   target.name = resolvePackageName(target.name.toString());
   target.version = !!target.version ? target.version : DEFAULT_VERSION;
-  target.specFileSuffix = normalizeToKebabOrSnakeCase(
+  target.specFileSuffix = normalizeToCase(
     options.specFileSuffix || 'spec',
+    'kebab-or-snake'
   );
 
+  target.caseNaming = !!target.caseNaming ? target.caseNaming : 'snake';
```

Comment:
Hi, maybe i missed something, but this shouldn't be `pascal` by default ?

Reply:
In the original implementation a case named `kebab-or-snake` is used, which is the `snake` you see referred here. I kept the original default to avoid unintended side effects (other piece of nestjs default to and expect `snake` as input, or they will complain)